frontend/src/pages/auth/auth.tsx

```diff
@@ -54,14 +55,28 @@ const Auth: React.FC = () => {
     [dispatch],
   );
 
+  const login = useGoogleLogin({
+    onSuccess: ({ code }) => {
+      void dispatch(authActions.loginWithGoogle({ code }));
+    },
+
+    flow: 'auth-code',
+  });
+  const handleGoogleLogin = useCallback(() => login(), [login]);
```

Comment:
Can't we pass this handler as it's returned from hook?
```suggestion
  const handleGoogleLogin = useGoogleLogin({
    onSuccess: ({ code }) => {
      void dispatch(authActions.loginWithGoogle({ code }));
    },

    flow: 'auth-code',
  });
```